modules/bullet/space_bullet.h

```diff
@@ -73,7 +73,6 @@ class BulletPhysicsDirectSpaceState : public PhysicsDirectSpaceState {
 	SpaceBullet *space;
 
 public:
-	BulletPhysicsDirectSpaceState(SpaceBullet *p_space);
 
```

Comment:
`clang-format` doesn't like the empty line after `public`: https://travis-ci.org/godotengine/godot/jobs/523390022